src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -86,7 +86,20 @@ public class PullRequestHandler : IHttpHandler
         uint pr;
         if (!string.IsNullOrEmpty(prString) && uint.TryParse(prString, out pr))
         {
-            success = SyncToPr(pr, result);
+            // Due to the timing of committing this PR service, syncing to low-number PRs will cause the script to break 
+            // Add a guard to prevent someone from hosing the PRService, but report success so we don't fail the PR service 
+            // PR numbers < 987 will never use this service  
+
+            if (pr < 987)
+            {
+                context.Response.Write(string.Format("PR specified, '{0}' is not supported by this script<br/>", HttpUtility.HtmlEncode(pr)));
+                context.Response.Write("Treating this as a no-op<br/>");
+                success = true;
+            }
+            else
+            {
+                success = SyncToPr(pr, result);
+            }
```

Comment:
It's easy to discover the URI for the service and hit the URI in a browser. 

It doesn't solve the problem entirely but it reduces (slightly) the ways in which someone could screw up the PRService. Other ways are being investigated too.